tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -63,19 +65,18 @@ spec:
       # end
   - name: run-loadtest
     image: 197575167141.dkr.ecr.us-west-2.amazonaws.com/clusterloader2:76e3fd7
-    command: ["./clusterloader"]
-    args: 
-    - --kubeconfig=$(workspaces.config.path)/kubeconfig
-    - --testconfig=$(workspaces.source.path)/perf-tests/clusterloader2/testing/load/config.yaml
-    - --testoverrides=$(workspaces.source.path)/overrides.yaml
-    - --nodes=$(params.nodes) 
-    - --provider=eks 
-    - --report-dir=$(workspaces.results.path)
-    - --alsologtostderr
-    env:
-    - name: ENABLE_EXEC_SERVICE
-      value: "false"
+    script: |
+      ENABLE_EXEC_SERVICE=false ./clusterloader --kubeconfig=$(workspaces.config.path)/kubeconfig --testconfig=$(workspaces.source.path)/perf-tests/clusterloader2/testing/load/config.yaml --testoverrides=$(workspaces.source.path)/overrides.yaml --nodes=$(params.nodes) --provider=eks --report-dir=$(workspaces.results.path) --alsologtostderr
```

Comment:
Do we need to pass `--provider=eks` when testing on KIT as well?

Reply:
Yes, `provider` is mandatory IIRC. Also, if we don't provide `eks` and provide something else, it will turn on other default features that works for other providers that we mention and we don't want to get into that business at this point, given our setup :D ( `eks` is safe bet because it asks for less feature-set in configuration)